pages/home.js

```diff
@@ -50,5 +52,34 @@ module.exports = {
   clickRemoveItems() {
     I.click(this.removeItems);
   },
+
+  async grabRemoveProductIcon() {
```

Comment:
don't create one line functions, if they are used only once in other function of same object